src/lib/libfs.js

```diff
@@ -983,7 +983,7 @@ FS.staticInit();`;
       var node = stream.node;
       var getattr = stream.stream_ops.getattr;
       var arg = getattr ? stream : node;
-      getattr ??= node.node_ops.getattr;
+      getattr ??= node.node_ops.getattr.bind(node.node_ops);
       FS.checkOpExists(getattr, {{{ cDefs.EPERM }}})
       return getattr(arg);
```

Comment:
How about `getattr.call(node.node_ops, arg)` here to avoid the temporary?

Reply:
The problem is that `getattr` might have come from either `stream.stream_ops` or `node.node_ops` and so the wrong `this` might break something else. I'm happy to rewrite the function, but I don't have enough experience with this project to test it thoroughly.